### PR TITLE
fix: i18next translation being displayed as unicode hex character

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes # mf-4091

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/mask/src/components/shared/PersonaSelectPanel/PersonaSelectPanel.tsx [hunk 1]
```diff
@@ -55,7 +55,16 @@ interface PersonaSelectPanelProps extends withClasses<'checked' | 'unchecked' |
 
 export const PersonaSelectPanel = memo<PersonaSelectPanelProps>((props) => {
     const { finishTarget, enableVerify = true, onClose } = props
-    const { t } = useI18N()
+
+    const { t: originalTran } = useI18N()
+    const t: typeof originalTran = useCallback(
+        (key, options = {}) => {
+            if (typeof options === 'string') return t(key, options)
```
Comment (guanbinrui) [inline packages/mask/src/components/shared/PersonaSelectPanel/PersonaSelectPanel.tsx]: Can we handle it in useI18N?

Comment (UncleBill) [reply packages/mask/src/components/shared/PersonaSelectPanel/PersonaSelectPanel.tsx]: @lelenei Learn unescape from https://www.i18next.com/translation-function/interpolation#unescape

**Review by UncleBill — CHANGES_REQUESTED** [review]

File: packages/mask/src/components/shared/PersonaSelectPanel/PersonaSelectPanel.tsx (hunk 1 above)
Comment (lelenei) [reply packages/mask/src/components/shared/PersonaSelectPanel/PersonaSelectPanel.tsx:62]: keyUnescaped does not seem to support

Comment (lelenei) [reply packages/mask/src/components/shared/PersonaSelectPanel/PersonaSelectPanel.tsx:62]: Unescaped not available

**Review by UncleBill — DISMISSED** [review]
Remove changes in tsx files.

**Comment by lelenei** [comment]
> Remove changes in tsx files.

**Review by UncleBill — APPROVED** [review]

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**